org-docs/CHARTER.md

```diff
@@ -22,6 +22,19 @@ MRTK3 is the next generation of MRTK, focused using OpenXR where applicable and
 Voting members may be added or removed by no less than 3/4 affirmative vote of the Steering Committee.
 The Steering Committee will appoint a Chair responsible for organizing Steering Committee activity.
 
+**2.2.1 Representation Guidelines*** Representation on the Steering Committee is evaluated by the sitting Steering Committee members based on the following guidelines. 
+
+**Platform Holders** Platform holders are alotted two voting representatives and meet the following criteria.
```

Comment:
The formatting needs to be cleaned up here.  The "Platform Holders" and "Technology Enables" look like the section headers.

Perhaps a table would best here

Reply:
Converted: 
![{CE4C58A5-7874-4DE5-96CB-FE1F58C134EF}](https://github.com/user-attachments/assets/e7089ead-aecb-4b4b-bac4-cff98b88f1c6)